tests/test-http-client-transport/src/test/java/org/eclipse/jetty/http/client/TransportScenario.java

```diff
@@ -328,8 +326,6 @@ public void startServer(Handler handler) throws Exception
         serverThreads.setName("server");
         serverThreads.setDetailedDump(true);
         server = new Server(serverThreads);
-        MBeanContainer mbeanContainer = new MBeanContainer(ManagementFactory.getPlatformMBeanServer());
-        server.addBean(mbeanContainer);
```

Comment:
Why was this removed?

Reply:
Because the tests using the `TransportScenario` were leaking resources when ran continuously. Since none of those seem to require the `MBeanContainer`, I decided to ditch it.

I have to re-investigate what made those resources leak and add a proper fix.

Reply:
There's no apparent leak anymore in the tests I tried. I'll add this back.